securedrop_client/models.py

```diff
@@ -26,9 +26,11 @@ def make_engine(home: str):
 
 class WithContent():
 
+    data = None
```

Comment:
Why is this a class variable? Shouldn't this be an instance variable?

Reply:
It's to ensure all instances have a `.data` field throughout their life-time - I was getting `instance has no attribute 'data'` at times otherwise.

Reply:
Ok so you should do something like 

```python
class WithContent:

    def __init__(self, *nargs, **kwargs) -> None:
        self.data = None
        super().__init__(*nargs, **kwargs)
```

This makes sure every instance has a `data` field. Though if a class with any fancy metaclasses subclasses this, it may break. But the solution you have will cause problems if more than one instance of this class is created.

Reply:
If it was a mutable value I could see how this pattern could cause issues. As it’s None, from where would the problems you see stem?

Reply:
On Python’s secretly prototypal object system - https://eev.ee/blog/2017/11/28/object-models/